scripts/datadog_wrapper.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
```

Comment:
Can we remove the `.sh` extension from the wrapper? the examples from AWS are all without the .sh extension, i think one benefit is that we could potentially change it to a python or non-sh script down the road without asking users to update their configurations.

Reply:
I agree it Should Be Fine to remove .sh as long as we have the shebang at the beginning of the file

Reply:
I will create a new revision for it.